app/views/pages/security.html.erb

```diff
@@ -0,0 +1,211 @@
+<% @title = "Security" %>
+
+<div class="l-colspan t-body">
+  <p>RubyGems.org Security</p>
+
+  <h2>Reporting a Rubygems or RubyGems.org security bug</h2>
+
+  <p>
+    For Rails, see <a href="http://rubyonrails.org/security/">Rails Security</a>
+  </p>
+
+  <h3>Getting a CVE</h3>
```

Comment:
https://github.com/RedHatProductSecurity/CVE-HOWTO is also a good howto for this as well.